.github/workflows/codeql-analysis.yml

```diff
@@ -16,10 +16,10 @@ jobs:
     - name: Checkout source
       uses: actions/checkout@v3
 
-    - name: Setup .NET Core SDK
-      uses: actions/setup-dotnet@v3
+    - name: Setup .NET SDK
+      uses: actions/setup-dotnet@v3.0.3
```

Comment:
Why pinning to 3.0.3?

Reply:
Why not?  Just updated as here https://github.com/graphql-dotnet/graphql-dotnet/blob/master/.github/workflows/codeql-analysis.yml#L20

Reply:
Because then you won’t get v3.0.4

Reply:
I don't understand. Dependabot bumps this version from time to time: 2 -> 3 -> 3.0.1 -> 3.0.2 -> 3.0.3

Reply:
That’s just the tag. “v3” is updated when there’s a new release. See here how the sha is the same for v3 and v3.0.3

https://github.com/actions/setup-dotnet/tags

Reply:
Ignore dependabot

Reply:
It means I don't know how versioning works in actions but I just did the same as we did in other repos.

Reply:
```suggestion
      uses: actions/setup-dotnet@v3
```